launcher_scripts/conf/peft/t5/squad.yaml

```diff
@@ -14,7 +14,7 @@ trainer:
   devices: 8
   accelerator: gpu
   num_nodes: 1
-  precision: bf16
+  precision: 16
```

Comment:
Is the precision meant to be 16 here?

Reply:
@thomasdhc Yes, it is meant to be 16. 
In our PEFT experiments, we didn't get to test thoroughly with bf16 yet, only with 16.